### PR TITLE
Cleans up /atom procs and defines

### PR DESCRIPTION
**Pull request by DominikPanic**
Removes unused procs and defines
Slightly cleans up and refactors existing one to /tg/ standard

### CODE REVIEW

File: code/game/atoms.dm [hunk 1]
```diff
@@ -332,45 +250,19 @@ its easier to just keep the beam vertical.
...
-			break
-//	to_chat(world, "X = [cur_x]; Y = [cur_y]")
-	if(cur_x && cur_y)
-		return list("x"=cur_x,"y"=cur_y)
-	else
-		return 0
-
 /atom/proc/checkpass(passflag)
-	return flags_pass&passflag
+	return CHECK_BITFIELD(flags_pass, passflag)
```
Comment (Rohesie) [inline code/game/atoms.dm]: This could be a macro instead. It's never redefined. Legacy /tg/ code.

File: code/game/atoms.dm [hunk 2]
```diff
@@ -76,8 +79,8 @@ directive is properly returned.
 /atom/proc/on_reagent_change()
 	return
 
-/atom/proc/Bumped(AM as mob|obj)
-	return
+/atom/proc/Bumped(atom/movable/AM)
+	set waitfor = FALSE
```
Comment (Rohesie) [inline code/game/atoms.dm]: While this is the /tg/ standard, I'm not entirely sure that's the best for us, as our bump code is not the same as theirs, nor theirs is necessarily the best here.

Comment (DominikPanic) [reply code/game/atoms.dm]: So what should I do

Comment (Rohesie) [reply code/game/atoms.dm]: Leave this one doing nothing, just returning.

**Pull request merged**